### DISCUSSION TITLE
Motion Controls Support

### DISCUSSION THREAD

**Discussion by MythicalPlayz** [skyline-emu/skyline#116] [Ideas]
I was think if it is possible to use the phones accelerometer as gyro conrols.
Thanks.And keep up the good work.

**Comment by RayTR8** [comment]
> I was think if it is possible to use the phones accelerometer as gyro conrols.
> Thanks.And keep up the good work.

That'd probably be further done the line when commercial games are supported, interesting idea but it'll be of no real use

**Comment by bylaws** [comment]
As king said, this is something we aim to do in the future after some games work so keep an eye out.

**Comment by sevketkrmsr** [comment]
Gpu support when ?

**Comment by RayTR8** [comment]
> Gpu support when ?

Please don't comment irrelevant questions on other issues

**Comment by willfaust** [comment]
> > I was think if it is possible to use the phones accelerometer as gyro conrols.
> > Thanks.And keep up the good work.
> 
> That'd probably be further done the line when commercial games are supported, interesting idea but it'll be of no real use

Why no real use? Motion controls are a pretty nice feature with switch games.

**Comment by PixelyIon** [comment]
~~This issue will be closed and subsequently deleted~~, this is already planned and will be implemented in time. We have much higher priority work at the moment.

**Comment by IvarWithoutBones** [comment]
Disregard the comment above, this is not a priority as of now but it is still a valid issue.
